README.md

```diff
@@ -65,7 +71,43 @@ $ (cd src/assets; npm install)
 $ (cd src/assets/athene2-editor; npm install)
 ```
 
-Now you can start the cluster:
+#### Troubleshooting
+##### Wrong php version
+If `php composer.phar install` fails with error
+```
+Your requirements could not be resolved to an installable set of packages.           
+ Problem 1
+ - doctrine/collections v1.4.0 requires php ^5.6 || ^7.0 -> your PHP version (5.5.9) does not satisfy that requirement.                                   
+ - doctrine/collections v1.4.0 requires php ^5.6 || ^7.0 -> your PHP version (5.5.9) does not satisfy that requirement.                                   
+ - Installation request for doctrine/collections v1.4.0 -> satisfiable by doctrine/collections[v1.4.0].                                           
+```
+then you can try updating php to 5.6:
+
+```sh
+sudo apt-get remove php5-cli
+sudo add-apt-repository ppa:ondrej/php
+sudo apt-get update
+sudo apt-get install php5.6
```

Comment:
Not sure, I used `sudo apt-get install php5.6`, because I followed this guide for updating https://joshtronic.com/2014/08/31/upgrade-to-php-56-on-ubuntu-1404-lts/
What is the difference in 5.6-cli?